apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -108,7 +109,7 @@ public void setup() {
 
     Calendar calen = Calendar.getInstance();
     calen.set(2021, 3, 17);
-    beneficiary.setLastUpdated(calen.getTime().toInstant());
+    beneficiary.setLastUpdated(calen.getTime().toInstant()); //
```

Comment:
```suggestion
    beneficiary.setLastUpdated(calen.getTime().toInstant());
```
Stray trailing comment here